applications/FluidDynamicsHydraulicsApplication/custom_utilities/hydraulic_fluid_auxiliary_utilities.cpp

```diff
@@ -307,4 +307,60 @@ void HydraulicFluidAuxiliaryUtilities::SetInletFreeSurface(ModelPart &rModelPart
     });
 }
 
+void HydraulicFluidAuxiliaryUtilities::CalculateArtificialViscosity(ModelPart &rModelPart,double WaterDynamicViscosityMax){
```

Comment:
```suggestion
void HydraulicFluidAuxiliaryUtilities::CalculateNonIntersectedElementsArtificialViscosity(
    ModelPart &rModelPart,
    double DynamicViscosityMax)
{
```
Note the name of the method and the argument to make clear what we are doing. Also note (again) our style guidelines.